src/terra-dev-site/doc/application/reference.1/components/ApplicationBase.doc.mdx

```diff
@@ -20,7 +20,7 @@ import ApplicationBase from 'terra-application/lib/application-base';
 
 ### I18n
 
-ApplicationBase exposes a prop from defining the application's locale. ApplicationBase renders an IntlProvider (from `react-intl`) around its children. Children can access intl APIs through the legacy Context API provided by `react-intl`.
+ApplicationBase exposes a prop for defining the application's locale. ApplicationBase renders an [IntlProvider](https://engineering.cerner.com/terra-ui/components/terra-i-18-n/i-18-n/i-18-n) (from [react-intl](https://github.com/formatjs/react-intl)) as well as a [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context).Provider around its children. Children can access the intl APIs by using the [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context)) or by using react-intl's provided components directly.
```

Comment:
I think it would be helpful to expand the description that the Intl Provider manages switching the local and loading new local data for the application. Just some context to why Base is providing them and what they are.

Reply:
> or by using react-intl's provided components directly.
 
probably feels redundant, but can we link to react-intl's wiki page here for quick ref since it's the first mention on this doc

Reply:
I rewrote this a bit to include your points.